internal/plugin/generate.go

```diff
@@ -57,6 +73,18 @@ func Generate(plugin *protogen.Plugin) error {
 				resources: rs,
 				messages:  ms,
 			}
+			f.services = append(f.services, generator)
+		}
+		result = append(result, f)
```

Comment:
I could do that, but I think its safe since we are doing a for-loop over `file.services`, [see](https://github.com/einride/protoc-gen-go-aip-test/blob/24aef3ba2c657f248f6b358f8423e0d32fba92b8/internal/plugin/generate.go#L83-L95)